emcc.py

```diff
@@ -2156,9 +2155,6 @@ def phase_linker_setup(options, state, newargs):
     # See: https://github.com/emscripten-core/emscripten/issues/12065
     # See: https://github.com/emscripten-core/emscripten/issues/12066
     settings.DYNCALLS = 1
-    settings.REQUIRED_EXPORTS += ['emscripten_stack_get_base',
-                                  'emscripten_stack_get_end',
-                                  'emscripten_stack_set_limits']
```

Comment:
Where can I find the js library deps for this one? (I see the diff in the Asyncify JS library adds one but not all three)

Reply:
The others are not needed AFAICT.  Maybe they were at some point in the past.